### DISCUSSION TITLE
[Feature] More powerful/alternative GPS picker

### DISCUSSION THREAD

**Discussion by nielstron** [immich-app/immich#12894] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When adding a location to an image/ changing the GPS location, I can search places using the search bar. However the current search engine is not very powerful and does not resolve locations beyond the precision of cities. It would be great to add a more powerful engine/let the user choose which engine to use to resolve to precise locations (streets, landmarks etc)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
The reverse geocoding is currently fairly rudimentary, we'll be adding more accurate methods in the future.

**Reply by nielstron** [reply]
This issue is not about reverse geocoding. It's about entering an address *manually* when trying to overwrite the GPS tag of an image.

**Reply by nielstron** [reply]
Please consider reopening the discussion

**Reply by alextran1502** [reply]
Are you talking about entering "124 street abc"?

**Reply by bo0tzz** [reply]
If so it's a duplicate of #8551

**Reply by nielstron** [reply]
Yes seems like this is the case. Thanks for clarifying
